bin/reth/src/config.rs

```diff
@@ -96,26 +96,26 @@ impl Default for TotalDifficultyConfig {
 /// Body stage configuration.
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct BodiesConfig {
-    /// The maximum number of bodies to download before committing progress to the database.
-    pub commit_threshold: u64,
-    /// The maximum number of bodies to request from a peer at a time.
-    pub downloader_batch_size: usize,
-    /// The number of times to retry downloading a set of bodies.
-    pub downloader_retries: usize,
-    /// The maximum number of body requests to have in flight at a time.
-    ///
-    /// The maximum number of bodies downloaded at the same time is `downloader_batch_size *
-    /// downloader_concurrency`.
-    pub downloader_concurrency: usize,
+    /// The batch size of non-empty blocks per one request
+    pub downloader_request_batch_size: usize,
```

Comment:
I've renamed this to `limit`, which follows the `limit` field of the `GetHeaders` request, and make it easier to distinguish from stream_batch_size